tests/e2e/specs/front-end-code.spec.js

```diff
@@ -41,5 +42,6 @@ describe( 'Front end code insertion', () => {
 		expect( content ).toContain( `<script data-parsely-site="e2etest.example.com" src="https://cdn.parsely.com/keys/e2etest.example.com/p.js?ver=${ PLUGIN_VERSION }" id="parsely-cfg"></script>` );
 		expect( content ).toContain( '<script id="wp-parsely-api-js-extra">' );
 		expect( content ).toContain( 'var wpParsely = {"apikey":"e2etest.example.com"};' );
+		expect( content ).toContain( '<script src="http://localhost:8889/wp-content/plugins/wp-parsely/build/init-api.js?ver=f68f5afd116c18f188eb32c9cd990d80" id="wp-parsely-api-js"></script>' );
```

Comment:
The `ver` will change when that script is re-built.  It will be tedious to update this each time.  Can we make this tolerant to a variable version string?